net/inadyn/Makefile

```diff
@@ -41,16 +41,13 @@ define Package/inadyn/conffiles
 /etc/inadyn.conf
 endef
 
-TARGET_CFLAGS += $(FPIC)
-
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--disable-static \
 	--enable-openssl \
-	--without-pic
+	--with-pic
```

Comment:
Why this change?

Reply:
Removed the CFLAGS hack and replaced with this.